src/module.ts

```diff
@@ -1434,6 +1434,14 @@ export class Module {
     binaryen._BinaryenSetOneCallerInlineMaxSize(size);
   }
 
+  getAllowHeavyweight(): bool {
+    return binaryen._BinaryenGetAllowHeavyweight();
+  }
+
+  setAllowHeavyweight(allowHeavyweight: bool): void {
+    binaryen._BinaryenSetAllowHeavyweight(allowHeavyweight);
+  }
+
```

Comment:
With recent changes this should renamed as:

```suggestion
  getAllowInliningFunctionsWithLoops(): bool {
    return binaryen._BinaryenGetAllowInliningFunctionsWithLoops();
  }

  setAllowInliningFunctionsWithLoops(enabled: bool): void {
    binaryen._BinaryenSetAllowInliningFunctionsWithLoops(enabled);
  }

```